### DISCUSSION TITLE
adding Kopp Brandet Inverters

### DISCUSSION THREAD

**Discussion by sleepymaxx** [nathanmarlor/foxess_modbus#451] [Ideas]
As in my last Question: is it possible to add Kopp Brandet (but in reality FoxESS) inverters to the code? Thanks in advance...

**Comment by canton7** [comment]
Didn't we do that in #431?

**Reply by sleepymaxx** [reply]
sorry my fault... the integration is working fine... but i read about the possibillity to ad a charging period.... and again i read later that this is´n possible due to your reply in here:
https://github.com/nathanmarlor/foxess_modbus_charge_period_card/discussions/12
 - so again sorry for bothering you

**Reply by canton7** [reply]
Yeah, the H3s only expose holding registers, not input registers, so there's no way to set anything.

**Reply by sleepymaxx** [reply]
my fault....
